src/components/inline-link/index.tsx

```diff
@@ -22,24 +22,24 @@ const InlineLink = ({
 	)
 
 	/**
-	 * This case is possible for some of the extra <a> elements that are added by
-	 * our `anchor-links` remark plugin.
+	 * An undefined "href" is possible for some of the extra "target" <a> elements
+	 * that are added by our `anchor-links` remark plugin. These elements have
+	 * an "id", and act as a target for anchor links, but do not have an href.
+	 *
+	 * For now, we still want to render these elements, as they're needed
+	 * for some anchor links to function.
 	 *
 	 * FUTURE TODOs
 	 *   - update the plugin to no longer add empty <a> elements
-	 *   - update this to throw an error
+	 *   - update this to throw an error if href is falsy
+	 *
+	 * Asana task:
+	 * https://app.asana.com/0/1100423001970639/1202488740132127/f
 	 */
-	const hasHref = Boolean(href)
-	if (!hasHref) {
-		if (IS_DEV) {
-			console.error('[InlineLink] a falsy `href` was given. Returning `null`.')
-		}
-
-		return null
-	}
+	const hrefWithFallback = href || '#'
```

Comment:
Expanded the comment here. This change allows `Link` to render without error. I think the proper long-term solution here would be to avoid generating `<a />` elements that do not have an `href` - but that feels outside the scope of this PR (would involve `anchor-link` plugin refactoring, I think).